zhaquirks/xiaomi/__init__.py

```diff
@@ -102,7 +102,6 @@
     descriptor_capability_field=0,
 )
 
-
```

Comment:
Re-add that line, so this file isn't changed

Reply:
Done